src/jail_rules.c

```diff
@@ -297,7 +298,7 @@ jail_rules_add_profile(
         }
         g_ptr_array_add(list, jail_rules_profile_new(path, require));
     } else {
-        GWARN("Profile %s doesn't exist", path);
+        GWARN("Profile %s doesn't exist / is not allowed", path ?: name);
```

Comment:
I would prefer separate warnings for absolute names, invalid suffixes and non-existent files.

Reply:
I'll shuffle it around a bit, improve diagnostics and relax name vs path portion to better accommodate existence checks before calling this func.